sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/TaskManagementSteps.java

```diff
@@ -53,5 +55,15 @@ public TaskManagementSteps(WebDriverHelpers webDriverHelpers) {
           webDriverHelpers.fillAndSubmitInWebElement(
               GENERAL_SEARCH_INPUT, EditCaseSteps.aCase.getUuid());
         });
+
+    When(
+        "^I am checking if the associated linked event appears in task management and click on it$",
+        () -> {
+          String eventUuid = apiState.getCreatedEvent().getUuid();
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(GENERAL_SEARCH_INPUT);
```

Comment:
Did you try to not use `waitUntilElementIsVisibleAndClickable`? Because these are already implemented in the `fillAndSubmitInWebElement` and `clickOnWebElementBySelector`.